application-accelerator/provenance.hbs.md

```diff
@@ -12,4 +16,20 @@ The `Provenance` transform provides traceability and visibility into the generat
 - When the application was bootstrapped
 - Who bootstrapped the application
 
-For more information on the structure of the file and how to enable application bootstrapping provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
\ No newline at end of file
+For more information on the structure of the file and how to enable application bootstrapping provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## Integration with AMR
+Application Accelerator also closely integrates with the [Artifact Metadata Repository](../scst-store/amr/overview.hbs.md) (AMR).
+
+Every time an application is generated with an accelerator, an event is sent to the AMR store that
+roughly captures the same set of information as described in the paragraph above.
+
+The relevant AMR data models are [AppAcceleratorRun](../scst-store/amr/data-model-and-concepts.hbs.md#appacceleratorruns) (one for each invocation, also includes version information
+about which accelerator was used) and [AppAcceleratorFragments](../scst-store/amr/data-model-and-concepts.hbs.md#appacceleratorfragments) (zero to N, tracks which fragments were used
```

Comment:
What does 'zero to N' mean here?
Can you put it in a sentence, it seems like it lacks context here?

Reply:
zero to N means there is between 0 and some number N instances of those, N being the number of fragments used by the accelerator. There is one instance of AppAcceleratorFragment for each (named) fragment used by the accelerator being run.

Hope that helps